assets/js/base/components/label/index.tsx

```diff
@@ -1,27 +1,29 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
-import { Fragment } from 'react';
+import { Fragment } from '@wordpress/element';
```

Comment:
We should make sure we're importing from `@wordpress/element` as much as possible (hence the change I made here in 8899541)